media-gfx/exiv2/files/exiv2-0.25-install-dirs.patch

```diff
@@ -0,0 +1,22 @@
+--- a/config/CMakeChecks.txt	2015-07-19 14:17:22.857046105 +0200
```

Comment:
The install-dirs patch will likely grow bigger anyway in an attempt to fix the libdir path and using GnuInstallDirs, such that a description will make sense.